src/spikeinterface/curation/train_manual_curation.py

```diff
@@ -0,0 +1,372 @@
+import os
+import pandas as pd
+import logging
+from joblib import Parallel, delayed
+import pickle as pkl
+from enum import Enum
+from sklearn.neural_network import MLPClassifier
+from skopt import BayesSearchCV
+from sklearn.experimental import enable_hist_gradient_boosting
+from sklearn.ensemble import HistGradientBoostingRegressor
+from sklearn.ensemble import RandomForestClassifier, AdaBoostClassifier, GradientBoostingClassifier
+from sklearn.svm import SVC
+from sklearn.linear_model import LogisticRegression
+from lightgbm import LGBMClassifier
+from catboost import CatBoostClassifier
+from xgboost import XGBClassifier
+from sklearn.pipeline import Pipeline
+
+# Configure logging
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+seed = 42
```

Comment:
Do we want to enforce static seed setting? Might be better to force the user to pass a seed